assets/package.json

```diff
@@ -81,6 +81,7 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.14.0",
+    "@rollup/plugin-alias": "^3.1.1",
```

Comment:
This should be in devDependencies since it is used during the build process.

Reply:
Easy fix. I'll take care of it.